app/js/controllers/main.js

```diff
@@ -20,7 +24,8 @@ export default class MainController extends BaseController {
 
     const mountNode = document.querySelector('.app-view-container');
     const speechController = new SpeechController();
-    const server = new Server();
+    const settings = new Settings();
+    const server = new Server({ settings });
```

Comment:
I didn't check for real but I checked the lib: settings.js actually listens for `storage` events to synchronize between several instances. I think this is especially useful if the user has several pages opened for the same app, but not really in the app itself.